app/services/ads.js

```diff
@@ -11,8 +11,11 @@ export default Service.extend({
   siteHeadOffset: 0,
   slotNames: null,
   noAdsQueryParam: null,
+  adsInMobileApp: null,
   noAds: computed('noAdsQueryParam', function () {
```

Comment:
computed('adsInMobileApp', 'adsInMobileApp' ...

Reply:
I got a comment about unfortunate name and while changing it I came up with an idea to remove `adsInMobileApp` from the service and in the application controller make a double alias! How good/bad is that? I'm pushing a code change in a minute and waiting for your advice ;)